plugin/fireplace.vim

```diff
@@ -1726,7 +1726,7 @@ function! s:RunTests(bang, count, ...) abort
     else
       let args = [fireplace#ns()]
       if a:count
-        let pattern = '^\s*(def\k*\s\+\(\h\k*\)'
+        let pattern = '^\s*(def\k*\s\+' . '\%(\^:\k+\s\+\)*' . '\(\h\k*\)'
```

Comment:
@tpope I splitted the pattern into 3 concatenated strings, because that way it was easier for me to understand it. But I can change it back to a single string if you like it that way better.